includes/class-wc-calypso-bridge-woocommerce-admin-features.php

```diff
@@ -77,7 +77,9 @@ public function initialize() {
 		 * @return array
 		 */
 		add_filter( 'pre_option_woocommerce_onboarding_profile', static function ( $option_value ) {
```

Comment:
The option name is different since now we're getting the actual value from database first

```suggestion
		add_filter( 'option_woocommerce_onboarding_profile', static function ( $option_value ) {
```